wordpress-mobile/WordPress-Android
# Revert app icons background update

This PR reverts the changes introduced by #14829.

~Since this targets the frozen branch, we should wait for confirmation before merging it.~ This can be merged as soon as it is approved.

#### To test:

Build the vanilla variant of the app and verify that the icon has its original background back. Try it out with different screen resolutions and Android versions.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
